.github/workflows/update-intellij-version.yml

```diff
@@ -0,0 +1,46 @@
+name: Update IntelliJ Version
+
+on:
+  pull_request:
+
+jobs:
+  bump-intellij-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Update default IntelliJ version
+        run: |
+          sbt "update-intellij / run --java-home /usr/local/openjdk-11"
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add .
```

Comment:
`git add .` and `git add -A .` are insecure in automated scenarios, as they might end up committing & pushing secret files. Probably not a concern for this specific CI pipeline in this specific repo, but better to use `git add -u .` (only stage modified & removed files, not new files) or `git commit -a ...`